HermaFx.Foundation/RuntimeExtensions/GenericExtensions.cs

```diff
@@ -135,5 +135,22 @@ public static T ThrowIfNull<T>(this T source, string paramName, string message,
 
 			return source;
 		}
+
+		public static string ThrowIfNullOrWhiteSpace(this string @this, string message, params object[] args)
```

Comment:
This is not a Generic extension, as this is for strings. Also this is more like a Guard, than an extension, so probably this should be placed at a new file for Guards (if there is not already an existing one).

Reply:
True, I should look another class like Guard.